### DISCUSSION TITLE
How to import a collection without changing paths?

### DISCUSSION THREAD

**Discussion by Ramblurr** [beetbox/beets#5062] [Q&A] [answered]
I have a large collection that I'd like to manage with beets. However because I already have other software that relies on those existing paths (for playlists and listening history), I'd like to import it while preserving the paths and only updating the metadata.

However for new music rips I'd like to use beets to name the files appropriately. 

Assume:

```
/data/music/old-library  # this is the old library with lots of music inside
/data/music/new-beets-library/ # This is the library in the beets config, it is currently empty
```

My plan is to import from `old-library` to `new-beets-library` then when the import is finished rename `new-beets-library` to `old-library` so my existing software can function as normal.

The problem is that the naming inside the old-library isn't 100% consistent, so I can't simply provide a `paths` config to beets to emulate the naming scheme in old-library.

Is there anyway to achieve this?

**Comment by JOJ0** [comment, marked as answer]
copy files to new location

then use beets import configured with

copy:no
move:no

for new imports use --copy cli option, paths config will be applied
